fe/src/main/java/org/apache/doris/common/FeConstants.java

```diff
@@ -31,6 +31,7 @@
     public static int shortkey_max_column_count = 3;
     public static int shortkey_maxsize_bytes = 36;
     public static long default_db_data_quota_bytes = 1024 * 1024 * 1024 * 1024L; // 1TB
+    public static long default_db_replica_quota_size = 1000000;
```

Comment:
The 1000000 is too small.  In meituan.com, the replica number for one database has exceeded 1000000. I think your initial value 1024 * 1024 * 1024 is OK.

Reply:
Why not set -1 as default value, which means no limit?
Otherwise I'm afraid some cluster can't create table after cluster upgrade.

Reply:
OK. I will change it to original default value

Reply:
@imay 1024 * 1024 * 1024 is a large value and enough, if one table has 100000 tablets , about 10000 tables can exits in on db?

Reply:
in my point of view, quota is a necessary limit for resource usage, the resource is limited, replica quota should like data quota, not set to be -1 @imay